README.md

```diff
@@ -80,6 +89,16 @@ Alternatively, you can just execute:
 ```bash
 sh ./scripts/presetup-rush.sh
 ```
+### Common Issues During Installation
+- **Permission Errors**: If you encounter permission issues, try running your terminal as an administrator or using `sudo` on Unix-based systems.
```

Comment:
I would not recommend write about sudo or administrative rights, since it is not required.